app/services/course/assessment/submission/statistics_download_service.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+require 'csv'
+class Course::Assessment::Submission::StatisticsDownloadService
+  include ApplicationFormattersHelper
+
+  class << self
+    # Downloads the statistics and zip them.
+    #
+    # @param [User] current_user The current user downloading the statistics.
+    # @param [Course::Assessment] assessment The assessments to download  from.
+    # @param [Array<Integer>] submission_ids The ids of the submissions to download statistics for
+    # @return [String] The path to the zip file.
+    def download(current_user, assessment, submission_ids)
```

Comment:
Actually if you wish to make the service even cleaner we can get `@assessment` from the `submission_ids`, and just retrieve the maximum grade. Might be easier when testing the service.

Reply:
OK, will remove ```assessment```